packages/flutter/test/widgets/image_test.dart

```diff
@@ -1915,6 +1905,11 @@ class TestImage implements ui.Image {
 
   @override
   String toString() => '[$width\u00D7$height]';
+
+  @override
```

Comment:
... and here.

Reply:
Refactored all of these, thanks for catching.